src/com/xilinx/rapidwright/util/function/InputStreamSupplier.java

```diff
@@ -49,7 +52,7 @@ static InputStreamSupplier fromPath(Path p) {
     public static InputStream getInputStream(Path fileName) {
         InputStream in = null;
         try {
-            if (fileName.toString().endsWith(".gz")) {
+            if (fileName.toString().endsWith(".gz") && Params.RW_DECOMPRESS_GZIPPED_EDIF) {
```

Comment:
Should another condition for decompressing to disk be that we are invoking the parallel EDIF parser?

Reply:
Both serial and parallel parsers are faster if decompressed to disk.  I think I treat them both the same.

Reply:
I did not know that. I retract!